modules/upgrade/pages/proxy-minor-uyuni.adoc

```diff
@@ -0,0 +1,37 @@
+[[proxy-package-uyuni]]
+= Proxy - Minor Upgrade
+
+Before you perform any proxy update, schedule a maintenance window.
+The clients registered to {productname} through the proxy will not be able to connect to {productname} while the update is in progress.
+For more information about maintenance windows, see xref:administration:maintenance-windows.adoc[].
+
+[NOTE]
+====
+Minor proxy upgrades do not include a version upgrade of the operating system.
+For example, while upgrading from 2021.06 to 2021.08 the operating system will stay at {opensuse} Leap 15.3.
+====
+
+
+
+== Upgrade the Proxy
+
+To update a proxy you first stop the proxy service, then update the software and finally restart the proxy service.
+
+
+
+.Procedure: Updating the {productname} Proxy
+. On the {productname} Proxy, stop the proxy service:
++
+----
+spacewalk-proxy stop
+----
+. In the {productname} Server {webui}, navigate to menu:Systems[Proxy] and click the name of the proxy.
+. Select packages to be updated on the proxy, and then apply the selection.
```

Comment:
As I user I'd ask: what packages?

And my reply would be `all of them`. Unless you have something else in mind.

Reply:
ok